gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/x509/X509AuthenticationUserDetailsService.groovy

```diff
@@ -52,6 +54,9 @@ class X509AuthenticationUserDetailsService implements AuthenticationUserDetailsS
 
   @Override
   UserDetails loadUserDetails(PreAuthenticatedAuthenticationToken token) throws UsernameNotFoundException {
+    log.debug("**** Entering loadUserDetails")
```

Comment:
I would these two debug lines, they were for our debugging process, but not useful outside the development of this feature.